bindings/ios/MEGARequest.h

```diff
@@ -471,6 +472,15 @@ typedef NS_ENUM (NSInteger, MEGANodeAccessLevel) {
  */
 @property (readonly, nonatomic) MEGAFolderInfo *megaFolderInfo;
 
+/**
+ *  @brief Returns settings for push notifications
+ *
+ * The SDK retains the ownership of the returned value. It will be valid until the MEGARequest object is deleted.
```

Comment:
```suggestion
```

This is internal for the SDK and transparent for Objective-C developers, we don't add this kind of information to the binding